### PR TITLE
Update docs on MSMQ bridge specifics

### PR DESCRIPTION
**Pull request by jpalac**
Update the documentation to:
- remove refence to `@machine` in the QueueAddress
- specify that the bridge needs to be installed on each MSMQ server

### CODE REVIEW

File: Snippets/Bridge/Bridge_2/Configuration.cs [hunk 1]
```diff
@@ -196,9 +196,9 @@ public void QueueName()
         #region custom-address
 
         var transport = new BridgeTransport(new MsmqTransport());
-        transport.HasEndpoint("Finance", "finance_queue_name");
+        transport.HasEndpoint("Finance", "finance@machinename");
```
Comment (andreasohlund) [inline Snippets/Bridge/Bridge_2/Configuration.cs:199]: I wonder if we should keep both examples:

1. Where the endpoint is on a different machine => finance@machinename
2. Where the queue name is different due to https://docs.particular.net/nservicebus/endpoints/specify-endpoint-name#input-queue being used? => finance_queue_name

Comment (andreasohlund) [reply Snippets/Bridge/Bridge_2/Configuration.cs:199]: And perhaps spell out that 1 is MSMQ specific and that 2 applies to all transports?

Comment (andreasohlund) [reply Snippets/Bridge/Bridge_2/Configuration.cs:199]: >Where the queue name is different due to https://docs.particular.net/nservicebus/endpoints/specify-endpoint-name#input-queue being used? => finance_queue_name

I've realized that my statement here is wrong, the address override should only be used to control where messages get dispatched to and where subscription messages are sent.

To control the name of the queue that the bridge reads from the endpoint name has to be adjusted since that is what we pass into the raw endpoint that does the reading.

https://github.com/Particular/NServiceBus.MessagingBridge/blob/master/src/NServiceBus.MessagingBridge/EndpointProxyFactory.cs#L28


In short: The text in this PR is ok even though I'm not sure if "machinename" is better than "another-machine"

Comment (jpalac) [reply Snippets/Bridge/Bridge_2/Configuration.cs:199]: I thought perhaps this PR is no longer relevant since we fixed it so that you can specify the machine name in the queue address, and we decided that just specifying a queue name isn't really supported?

Comment (andreasohlund) [reply Snippets/Bridge/Bridge_2/Configuration.cs:199]: Got it, so we can close this PR
